dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/HibernateEventChangeLogStore.java

```diff
@@ -159,14 +166,21 @@ public void deleteEventChangeLog(Event event) {
     entityManager.createQuery(hql).setParameter("event", event).executeUpdate();
   }
 
-  private static String sortExpressions(List<Order> order) {
-    if (order.isEmpty()) {
+  private static String sortExpressions(Order order) {
```

Comment:
Any particular reason why this is `static`?

Reply:
Why should it not be? 🤔 Functions should be `static`.